src/scss/components/search/SearchFacade.scss

```diff
@@ -6,30 +6,38 @@
 
 @mixin SearchFacadePristineFocus {
   border: 1px solid $cyan-700;
```

Comment:
Please use updated  `v2.0` colors to replace `$cyan-XXX`.

Reply:
Changes have been done, Please review